ci/git-repo-fetch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+dir=${1}
+refspec=${2:-master}
+repo=${3:-projectriff/fats}
+
+if [ ! -f $dir ]; then
+  mkdir -p $dir
+  curl -L https://github.com/${repo}/archive/${refspec}.tar.gz | \
```

Comment:
this isn't really fetching a git repo. It's fetching a tgz from GitHub that happens to be a repo. Other than the name GitHub, there is no git, nor git fetch.